packages/graphiql-react/package.json

```diff
@@ -11,8 +11,19 @@
     "url": "https://github.com/graphql/graphiql/issues?q=issue+label:@graphiql/react"
   },
   "license": "MIT",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.es.js",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js",
+      "types": "./types/index.d.ts"
+    },
+    "./font/roboto.css": "./font/roboto.css",
+    "./font/fira-code.css": "./font/fira-code.css",
+    "./dist/style.css": "./dist/style.css"
+  },
```

Comment:
without these `exports`, I get the following errors from `graphiql`

> It happens because starting from vite v3 it compile entry to `js`/`mjs` e.g. `index.js` and `index.mjs`

<img width="700" alt="image" src="https://github.com/graphql/graphiql/assets/7361780/cd3da84c-9116-474b-a641-0385a41e4a25">


```sh
Compiled with problems:X

WARNING in ../../../node_modules/@radix-ui/react-arrow/dist/index.mjs 13:68-104

export 'default' (imported as '$jbnEx$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dialog/dist/index.mjs 75:71-107

export 'default' (imported as '$67UHm$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dialog/dist/index.mjs 121:83-119

export 'default' (imported as '$67UHm$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dialog/dist/index.mjs 139:64-100

export 'default' (imported as '$67UHm$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dialog/dist/index.mjs 159:100-136

export 'default' (imported as '$67UHm$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dialog/dist/index.mjs 161:90-126

export 'default' (imported as '$67UHm$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dialog/dist/index.mjs 176:87-123

export 'default' (imported as '$67UHm$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dialog/dist/index.mjs 202:87-123

export 'default' (imported as '$67UHm$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dialog/dist/index.mjs 249:67-103

export 'default' (imported as '$67UHm$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dialog/dist/index.mjs 266:67-103

export 'default' (imported as '$67UHm$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dialog/dist/index.mjs 281:66-102

export 'default' (imported as '$67UHm$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dialog/dist/index.mjs 296:71-107

export 'default' (imported as '$67UHm$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dismissable-layer/dist/index.mjs 119:68-104

export 'default' (imported as '$kqwpH$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dismissable-layer/dist/index.mjs 151:68-104

export 'default' (imported as '$kqwpH$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 51:55-91

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 68:61-97

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 70:79-115

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 115:61-97

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 128:62-98

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 164:60-96

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 177:60-96

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 190:59-95

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 203:67-103

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 216:65-101

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 229:64-100

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 242:68-104

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 255:64-100

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 268:60-96

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 285:58-94

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 296:65-101

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-dropdown-menu/dist/index.mjs 309:65-101

export 'default' (imported as '$9kmUS$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-focus-scope/dist/index.mjs 163:68-104

export 'default' (imported as '$45QHv$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 157:61-97

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 201:106-142

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 203:92-128

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 215:85-121

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 232:85-121

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 341:56-92

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 353:59-95

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 414:68-104

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 428:68-104

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 462:82-118

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 517:55-91

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 521:65-101

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 554:85-121

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 583:85-121

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 600:85-121

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 628:65-101

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 641:68-104

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 657:60-96

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 731:89-125

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 733:86-122

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-menu/dist/index.mjs 844:81-117

export 'default' (imported as '$epM9y$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-popper/dist/index.mjs 64:88-124

export 'default' (imported as '$kY93V$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-popper/dist/index.mjs 190:64-100

export 'default' (imported as '$kY93V$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-popper/dist/index.mjs 245:55-91

export 'default' (imported as '$kY93V$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-portal/dist/index.mjs 16:123-159

export 'default' (imported as '$7SXl2$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-primitive/dist/index.mjs 39:56-92

export 'default' (imported as '$4q5Fq$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-roving-focus/dist/index.mjs 42:86-122

export 'default' (imported as '$98Iye$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-roving-focus/dist/index.mjs 92:64-100

export 'default' (imported as '$98Iye$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-roving-focus/dist/index.mjs 163:65-101

export 'default' (imported as '$98Iye$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-slot/dist/index.mjs 25:83-119

export 'default' (imported as '$9IrjX$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-slot/dist/index.mjs 29:79-115

export 'default' (imported as '$9IrjX$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-tooltip/dist/index.mjs 185:61-97

export 'default' (imported as '$8wepK$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-tooltip/dist/index.mjs 187:81-117

export 'default' (imported as '$8wepK$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-tooltip/dist/index.mjs 252:118-154

export 'default' (imported as '$8wepK$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-tooltip/dist/index.mjs 256:92-128

export 'default' (imported as '$8wepK$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-tooltip/dist/index.mjs 347:88-124

export 'default' (imported as '$8wepK$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-tooltip/dist/index.mjs 392:58-94

export 'default' (imported as '$8wepK$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-tooltip/dist/index.mjs 423:107-143

export 'default' (imported as '$8wepK$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


WARNING in ../../../node_modules/@radix-ui/react-visually-hidden/dist/index.mjs 12:69-105

export 'default' (imported as '$kVwnw$babelruntimehelpersesmextends') was not found in '@babel/runtime/helpers/esm/extends' (module has no exports)


ERROR in ../../graphiql-react/dist/index.mjs 1170:4-6

export 'Combobox' (imported as 'de') was not found in '@headlessui/react' (module has no exports)


ERROR in ../../graphiql-react/dist/index.mjs 1189:16-24

export 'Combobox' (imported as 'de') was not found in '@headlessui/react' (module has no exports)


ERROR in ../../graphiql-react/dist/index.mjs 1204:31-41

export 'Combobox' (imported as 'de') was not found in '@headlessui/react' (module has no exports)


ERROR in ../../graphiql-react/dist/index.mjs 1206:12-21

export 'Combobox' (imported as 'de') was not found in '@headlessui/react' (module has no exports)


ERROR in ../../graphiql-react/dist/index.mjs 1216:12-21

export 'Combobox' (imported as 'de') was not found in '@headlessui/react' (module has no exports)


ERROR in ../../graphiql-react/dist/index.mjs 1225:12-21

export 'Combobox' (imported as 'de') was not found in '@headlessui/react' (module has no exports)
```